pkg/apis/pxc/v1/pxc_prestore_types.go

```diff
@@ -27,6 +27,7 @@ type PITR struct {
 	Type         string           `json:"type"`
 	Date         string           `json:"date"`
 	GTIDSet      string           `json:"gtidSet"`
+	GTID         string           `json:"gtid"`
```

Comment:
if we decided to add new field we need to add it to `deploy/backup/restore.yaml` too

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/801/commits/1366152419a43bc8f62c50ee14d339469751fafc